symphony/bdk/core/auth/auth_session.py

```diff
@@ -0,0 +1,34 @@
+class AuthSessionRSA:
+    """
+    RSA Authentication session handle.
+    """
+
+    def __init__(self, authenticator):
+        self.__session_token = None
+        self.__key_manager_token = None
+        self.__authenticator = authenticator
+
+    def get_session_token(self) -> str:
+        """
+        Get the pod's authentication token.
+
+        Returns: the Pod session token.
```

Comment:
Please use [reStructuredTex](https://docutils.sourceforge.io/rst.html) as we will use Sphynx to generate documentation. It is also the standard advocated by [PEP-287](https://www.python.org/dev/peps/pep-0287/)